c4p2n/models.py

```diff
@@ -62,6 +74,26 @@ class CallForPaperFormResponse(BaseModel):
     answers: List[Union[TextAnswer, MultipleChoiceAnswer]]
 
 
+class CallForPaperAnswers(BaseModel):
```

Comment:
What about renaming to `CallForPaperApplication` or something without plural `answers`